models.py

```diff
@@ -29,3 +29,14 @@ class EdxNodeBBCategory(models.Model):
 
     def __str__(self):
         return str(self.course_key)
+
+
+class EdxNodeBBEnrollment(models.Model):
```

Comment:
Why have you named this `EdxNodeBBEnrollment` when  you are only saving only edx related data. Besides, I think this data can also be retrieved from  edx current db schema.

Reply:
This data is saved already in edx database but we have created a new table just to be sure that the user is successfully joined in a group of that course on nodebb. We save into this table when we get OK response from nodebb join group call.

We can make changes here by changing the field from edx_user to `nodebb_uid` and `course_id` to nodebb `group_slug` but that doesn't make any changes I think. in both cases, we are saving data to make sure our edx and nodebb are completely synced.

Reply:
I get your point but this class name is confusing.